apps/webClient/src/views/Home/index.tsx

```diff
@@ -56,47 +56,21 @@ const Home: React.FC<Props> = ({}) => {
                   sollicitudin aliquam ultrices sagittis.
                 </i>
               </Typography>
+              <Box mt={2}>
+                <Grid container justify="center">
```

Comment:
I think this should be a `Grid container item` as we should have at least one item in the grid.